sway-core/tests/sway_to_ir/mutable_struct.ir

```diff
@@ -0,0 +1,31 @@
+script {
+    fn main() -> u64, !1 {
+        local mut ptr { u64, u64 } record
+
+        entry:
+        v0 = const { u64, u64 } { u64 undef, u64 undef }, !2
+        v1 = const u64 40, !3
+        v2 = insert_value v0, { u64, u64 }, v1, 0, !2
+        v3 = const u64 2, !4
+        v4 = insert_value v2, { u64, u64 }, v3, 1, !2
+        v5 = get_ptr mut ptr { u64, u64 } record, ptr { u64, u64 }, 0, !5
```

Comment:
It looks like you're re-instating these files when they should be deleted?  Or am I reading this wrongly?  The `sway_to_ir` directory shouldn't exist on `master` any more.

Reply:
Good eye---I didn't realize that the merge added these files back in.